src/ConnectedMode/ConnectedMode.csproj

```diff
@@ -16,6 +16,7 @@
     <AssemblyName>SonarLint.VisualStudio.ConnectedMode</AssemblyName>
     <RootNamespace>SonarLint.VisualStudio.ConnectedMode</RootNamespace>
     <RequiresLibGit>true</RequiresLibGit>
+    <RequiresNewtonsoft>true</RequiresNewtonsoft>
```

Comment:
This tells our import targets to import the correct reference version of the NewtonSoft NuGet package.

The solution actually builds without this explicit reference, but that's just luck (`ConnectedMode` depends on something else that uses the NewtonSoft assemblies, so it will be copied during the build). It's better to explicit.